release-procedure.md

```diff
@@ -21,7 +21,7 @@ $ git checkout -b release-X.Y upstream/release-X.Y
 $ git tag -s -m "Release for cloud-provider-openstack to support Kubernetes release x" vX.Y.Z
 $ git push upstream vX.Y.Z
 ```
-3. Build and push images to dockerhub
+3. Build and push images to dockerhub. Provide `ALL_ARCH` if needs to push only for specific types of architecture (for example `ALL_ARCH='arm64v8 amd64'`).
```

Comment:
sorry, silly question may be..ALL_ARCH is environment variable we have to set?

Reply:
Got to change `ALL_ARCH` here to `ARCHS`:)
Also you need to set this only if you plan to run a specific set of Architectures
Default will be `amd64 arm32v7 arm64v8 ppc64le s390x` if not set this environment variable.